### DISCUSSION TITLE
90° Entrances/Exits

### DISCUSSION THREAD

**Discussion by That-Human-Being** [OpenRCT2/OpenRCT2#19198] [Feature requests] [answered]
I created demo images based upon the Plain Entrance and Exit sprites in `g1.dat` and could in a few days make similar sprites for all the base-game entrance types, I am really just asking if someone could implement support for this feature in the game

![Plain-En-TL](https://user-images.githubusercontent.com/81326222/213009756-db97a9b7-46cf-4f7d-a389-19199cceda4d.png) ![Plain-En-TR](https://user-images.githubusercontent.com/81326222/213009758-3e5a626b-4726-4ccd-b34a-68ac49800787.png) ![Plain-En-TB](https://user-images.githubusercontent.com/81326222/213009755-2f56e957-d514-4b2c-9c85-e05038b827b4.png) ![Plain-Ex-TL](https://user-images.githubusercontent.com/81326222/213009761-1bfdc723-d098-4c62-8fbd-2b35477ac33d.png) ![Plain-Ex-TR](https://user-images.githubusercontent.com/81326222/213009763-3b593cc3-a3a9-4967-87aa-7ccb0d6a231d.png) ![Plain-Ex-TB](https://user-images.githubusercontent.com/81326222/213009760-8543b446-5a61-4ac3-98a4-735786a776ec.png)
The missing parts of the Entrances/Exits are separate sprites that I believe can be reused

**Comment by ocalhoun6** [comment]
If this is to be implemented as an officially supported feature, a really nice bonus feature to have would be for the entrance/exit to *automatically* turn to face the first path placed next to it. (The first queue path for the entrance, the first normal path for the exit.) So then it wouldn't even require using the tile inspector.

(Of course, that's something you probably wouldn't do in the initial PR, probably something you'd want to add in an additional PR afterward.)

------

Also ... how does this work with the abstract entrance style? Will the transparency be an extra challenge there?

**Reply by That-Human-Being** [reply]
About the Abstract Entrances: I'll probably figure that out next because that certainly would provide the most challenge

**Comment by That-Human-Being** [comment]
The Abstract Entrances, I'll do the Exits next
![Abstrast Entrance](https://user-images.githubusercontent.com/81326222/213540856-c15adce6-b45e-4547-b253-95ab7df52093.png) ![Abstrast Entrance-Frames](https://user-images.githubusercontent.com/81326222/213540860-037aae13-9b2c-4abb-9cc6-2b2266c936fe.png) ![Abstrast Entrance-Glass](https://user-images.githubusercontent.com/81326222/213540861-628ee047-2abb-4d0e-a2b2-e74aff03fc31.png)

**Comment by J-PIE-314** [comment, marked as answer]
This is the same request as #15704
